pyk4a/config.py

```diff
@@ -10,11 +10,16 @@ class FPS(IntEnum):
 
 
 # k4a_image_format_t
-class ColorFormat(IntEnum):
-    MJPG = 0
-    NV12 = 1
-    YUY2 = 2
-    BGRA32 = 3
+class ImageFormat(IntEnum):
```

Comment:
Renamed according C structure.
ImageFormat  can be used for recording/playback track information